addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I'm fine with the

```
const {Component, computed, run} = Ember
const {readOnly} = computed
```
style

@juwara0 can we align on an approach for single versus multi-line?  If so, I'll put this out to #frost-design as a new convention and close the associated issue